### ISSUE TITLE
Better look and behaviour of health and experience bars

### ISSUE THREAD

**Issue by Soszu** [Soszu/TheChroniclesOfAndaria#23]
Probably reimplementation of bar widget class will be necessary.

**Comment by szreder** [comment]
I could tackle this issue. Have you got any ideas, mockups or whatever regarding the proper look and behaviour of such bar widgets? Any significant and worth mentioning feature requests?

**Comment by Soszu** [comment]
You're at liberty here. As for me it would be nice if health bar will also give information about regeneration, but if you want to do it another way - feel free to do so. 

**My proposal:**  

Something similar to:
http://i437.photobucket.com/albums/qq99/masternm34/health.jpg
http://i437.photobucket.com/albums/qq99/masternm34/actionbar.jpg
_Circle on the left can be used for displaying current character's avatar._

**To sum up** there are few things to be displayed (not necessarily all of them in the same widget)
- avatar
- health
- regeneration
- experience
- reputation*

*another widget for this may be a good idea  

**This also may be inspiring:**
http://2.bp.blogspot.com/-I9hBx1xgDU0/UGLajyIjH2I/AAAAAAAAAcg/wqYSrh0JSsw/s640/Tri-Agent+Health+Concepts.png
